xapian-letor/include/xapian-letor/ranker.h

```diff
@@ -354,6 +354,69 @@ class XAPIAN_VISIBILITY_DEFAULT SVMRanker: public Ranker {
 
 };
 
+class XAPIAN_VISIBILITY_DEFAULT ListMLERanker : public Ranker {
+    /// Ranker parameters
+    std::vector<double> parameters;
+    /// Learning rate (Default is 0.001)
+    double learning_rate;
+    /// Number of iterations (Default is 10)
+    int iterations;
+
+    /** Train the model.
+     *
+     * @exception InvalidArgumentError will be thrown if training_data is
+     *            empty.
+     */
+    void train(const std::vector<Xapian::FeatureVector>& training_data);
+
+    /** Save ListMLE model as db metadata.
+     *
+     *  ListMLE model gets stored with serialised value of each parameter in
+     *  a new line.
```

Comment:
You've just done `s/each parameter value/serialised value of each parameter/`here, which means it's now not correct for either how the code used to be, nor how it is now - there are no newlines between serialised values (as you can clearly see if you look at the implementation).

And the example still shows how this might have been before we changed to using `serialise_double()`.

As I said, I don't think details of the serialisation belong here, but we definitely don't want **incorrect** details of the serialisation here.

I'd just say "The model is serialised as a binary blob."  That tells the API user that printing it out isn't going to be useful, or at least if they do, then seeing binary data doesn't indicate a problem.  Beyond that, they really don't need to worry about the format of the serialisation.  By analogy, we don't document the format of the serialisation returned by e.g. `Document::serialise()` in its doc comment.